.eslintrc.json

```diff
@@ -1,7 +1,8 @@
 {
   "env": {
     "browser": true,
-    "es2021": true
+    "es2021": true,
+    "node": true
```

Comment:
I needed to add `node` here to get lint to stop whining about `process` not being accessible in `docs-theme.mjs`.